### ISSUE TITLE
Feature request: Only check staged files

### ISSUE THREAD

**Issue by ssalka** [cschroeter/ts-strictify#3]
When having a lot of changed files with the intent of making several small commits, ts-strictify will check all changed files, when it really only should check the files I have staged via `git add`.

**Comment by cschroeter** [comment]
Mhh so at this moment `ts-strictify` will check for local changes (git status) and files that have been modified since your local branch was forked from the target branch. 

While the first option is aimed for developers to check their local changes, the second check is for CI tools to verify that all files in the feature branch are strict compliant. Maybe we can have

**Comment by cschroeter** [comment]
Ah, as a workaround you could use a non existing target branch

```
➜ yarn ts-strictify --targetBranch nope
yarn run v1.17.3
⚠️  Can not find commit at which the current branch was forked from nope. Does target branch nope exists?
🎯  Found 0 changed files
 All files passed
✨  Done in 0.43s.
```

**Comment by ssalka** [comment]
Hmm, the simplest way would be to allow an additional option, e.g. `--trackedFilesOnly` or `--noUntracked`, or something more general like `--statusTypes="created,modified"`, which could default to the existing behavior when that flag is not passed.

Either of these could be given as an argument to `findModifiedAndUntrackedFiles`, which would then selectively pick from the results of `simpleGit().status()`

**Comment by miluoshi** [comment]
Running:
```
npx ts-strictify --targetBranch `git rev-parse --abbrev-ref HEAD`
```
will check the code against the current branch, which is closest option to checking only staged files

**Comment by cschroeter** [comment]
Available in 1.4.0
